response_operations_ui/controllers/notify_controller.py

```diff
@@ -39,16 +39,19 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
             notification.update({"reference": reference})
 
         url = f'{self.notify_url}{template_id}'
-
         auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
         response = requests.post(url, auth=auth, json=notification)
-        if response.status_code == 201:
+        status_code = response.status_code
+
+        try:
+            response.raise_for_status()
             logger.info('Notification id sent via Notify-Gateway to GOV.UK Notify.', id=response.json().get("id"))
-        else:
+        except HTTPError:
             ref = reference if reference else 'reference_unknown'
-            raise NotifyError("There was a problem sending a notification to Notify-Gateway to GOV.UK Notify",
-                              reference=ref)
-
+            logger.error('There was a problem sending a notification to Notify-Gateway', 
+                         ref=ref, status_code=status_code, message=response.text)
+            raise NotifyError()        
```

Comment:
You can't and shouldn't leave this empty.  If you see what other stuff in  the code handles these errors,  you'll notice that they all have `msg=e.description` in the log lines.  You should at the very minimum change this line to 
`raise NotifyError('There was a problem sending a notification to Notify-Gateway') and tbh it should probably be 
`raise NotifyError('There was a problem sending a notification to Notify-Gateway', error=response.text) so that if other code wants to query what went wrong, then it can.

Honestly I think this needs looking at properly down the line, but what you're doing here is an improvement.